Client/Frontend/Theme/LegacyThemeManager/ThemedWidgets.swift

```diff
@@ -93,6 +93,45 @@ class ThemedTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable {
     func configure(viewModel: ThemedTableViewCellViewModel) {
         self.viewModel = viewModel
     }
+
+    func restoreToInitialState() {
+        textLabel?.text = nil
+        textLabel?.font = DynamicFontHelper.defaultHelper.DefaultStandardFont
+        detailTextLabel?.text = nil
+        accessoryView = nil
+        accessoryType = .none
+        editingAccessoryView = nil
+        editingAccessoryType = .none
+        imageView?.image = nil
+        contentView.subviews.forEach {
+            $0.removeFromSuperview()
+        }
+        subviews.forEach {
+            $0.alpha = 1.0
+        }
+    }
+}
+
+/// Themed Table View Cell with cell style type subtitle
+class ThemedSubtitleTableViewCell: ThemedTableViewCell {
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Themed Table View Cell with cell style type value1
+class ThemedValue1TableViewCell: ThemedTableViewCell {
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .value1, reuseIdentifier: reuseIdentifier)
```

Comment:
Value1 means text is [left aligned](https://developer.apple.com/documentation/uikit/uitableviewcell/cellstyle/value1), so I would probably name this `ThemedLeftAlignedTableViewCell` instead to make it more intuitive